NPCs/Hostiles/SacSpiderFloored.cs

```diff
@@ -70,7 +70,7 @@ public override void AI()
                 {
                     for (int j = tileCoordY - 1; j <= tileCoordY + 1; j++)
                     {
-                        if (Framing.GetTileSafely(i, j).wall > WallID.None)
+                        if (Main.tile[i, j].wall > 0)
```

Comment:
Avoid magic numbers.

Reply:
Furthermore, why was this changed from `Framing.GetTileSafely` to directly checking the `Main.tile` array?